### PR TITLE
Address package FIXME comments in Neutron roles

### PR DESCRIPTION
**Pull request by channus**
TLDR; yes, we need all these packages.

### CODE REVIEW

File: roles/neutron-common/tasks/main.yml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 ---
-  # FIXME Do we need this pkg on rhel?
```
Comment (channus) [inline roles/neutron-common/tasks/main.yml]: Yes we need brctl

File: roles/neutron-data-network/tasks/main.yml [hunk 2]
```diff
@@ -1,12 +1,10 @@
 ---
-  #FIXME Do we need these pkgs on Rhel?
 - name: install neutron-data-network packages
   package: name={{ item }}
   with_items:
     - dnsmasq
     - dnsmasq-utils
     - ucarp
-    - radvd
```
Comment (channus) [inline roles/neutron-data-network/tasks/main.yml]: radvd used for ipv6 so not needed

File: roles/neutron-data-network/tasks/main.yml [hunk 3]
```diff
@@ -1,12 +1,10 @@
 ---
-  #FIXME Do we need these pkgs on Rhel?
```
Comment (channus) [inline roles/neutron-data-network/tasks/main.yml]: dnsmasq is used by the dhcp agents and we use ucarp for HA

File: roles/neutron-data-network/tasks/main.yml (hunk 2 above)
Comment (channus) [inline roles/neutron-data-network/tasks/main.yml]: Nevermind, even though we don't use ipv6 Neutron still requires it for HA Routers.

File: roles/neutron-data/tasks/main.yml [hunk 4]
```diff
@@ -1,5 +1,4 @@
 ---
-  #FIXME do we need this on rhel?
```
Comment (channus) [inline roles/neutron-data/tasks/main.yml]: Yes.  bwm-ng is a backend for sysctl, disk, netstat and multiple other tools.
ipset is used on compute nodes by neutron.
ebtables is used by neutron

File: roles/neutron-data-network/tasks/main.yml (hunk 3 above)
Comment (channus) [inline roles/neutron-data-network/tasks/main.yml]: radvd is used by neutron for router HA, even though we dont use ipv6

File: roles/neutron-data-network/tasks/main.yml [hunk 5]
```diff
@@ -27,7 +26,6 @@
     name: iputils
   when: os == 'rhel'
 
-#FIXME Do we need this for RHEL?
```
Comment (channus) [inline roles/neutron-data-network/tasks/main.yml]: yes we need keepalived for l3 router HA

**Comment by nirajdp76** [comment]
retest

**Pull request merged**